burrow/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,10 @@
+name: burrow
+files:
+- name: burrow.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
```

Comment:
Just a note that the service option provided here is only going to work with a recent agent 6